2024-12-19T19:22:09.226Z [ERROR]: Error: Route not found
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 404
}
Stack trace:
Not Found: Route not found
    at notFoundMiddleware (\middlewares\notFoundMiddleware.ts:10:7)
2024-12-19T19:22:32.549Z [ERROR]: Error: Route not found
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 404
}
Stack trace:
Not Found: Route not found
    at notFoundMiddleware (\middlewares\notFoundMiddleware.ts:10:7)
2024-12-19T19:22:33.960Z [ERROR]: Error: Route not found
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 404
}
Stack trace:
Not Found: Route not found
    at notFoundMiddleware (\middlewares\notFoundMiddleware.ts:10:7)
2024-12-19T19:22:51.116Z [ERROR]: Error: Route not found
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 404
}
Stack trace:
Not Found: Route not found
    at notFoundMiddleware (\middlewares\notFoundMiddleware.ts:10:7)
2024-12-19T19:22:52.411Z [ERROR]: Error: Route not found
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 404
}
Stack trace:
Not Found: Route not found
    at notFoundMiddleware (\middlewares\notFoundMiddleware.ts:10:7)
2024-12-19T19:23:29.387Z [ERROR]: Error: Invalid username or password.
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 401
}
Stack trace:
Unauthorized: Invalid username or password.
    at AuthService.login (\services\authService.ts:105:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async AuthController.login (\controllers\authController.ts:31:40)
2024-12-19T19:23:30.704Z [ERROR]: Error: Invalid username or password.
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 401
}
Stack trace:
Unauthorized: Invalid username or password.
    at AuthService.login (\services\authService.ts:105:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async AuthController.login (\controllers\authController.ts:31:40)
2024-12-19T19:23:31.790Z [ERROR]: Error: Account is locked. try again later.
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 429
}
Stack trace:
Too Many Requests Error: Account is locked. try again later.
    at \utils\limiter.ts:71:12
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
2024-12-19T19:23:32.627Z [ERROR]: Error: Account is locked. try again later.
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 429
}
Stack trace:
Too Many Requests Error: Account is locked. try again later.
    at \utils\limiter.ts:71:12
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
2024-12-19T19:27:51.194Z [ERROR]: Error: Account is locked. try again later.
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 429
}
Stack trace:
Too Many Requests Error: Account is locked. try again later.
    at \utils\limiter.ts:71:12
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
2024-12-19T19:27:57.986Z [ERROR]: Error: Account is locked. try again later.
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 429
}
Stack trace:
Too Many Requests Error: Account is locked. try again later.
    at \utils\limiter.ts:47:11
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
2024-12-19T19:28:04.776Z [ERROR]: Error: Account is locked. try again later.
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 429
}
Stack trace:
Too Many Requests Error: Account is locked. try again later.
    at \utils\limiter.ts:47:11
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
2024-12-19T19:28:14.462Z [ERROR]: Error: Account is locked. try again later.
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 429
}
Stack trace:
Too Many Requests Error: Account is locked. try again later.
    at \utils\limiter.ts:47:11
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
2024-12-19T19:28:20.739Z [ERROR]: Error: Account is locked. try again later.
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 429
}
Stack trace:
Too Many Requests Error: Account is locked. try again later.
    at \utils\limiter.ts:71:12
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
2024-12-19T19:28:55.862Z [ERROR]: Error: Invalid username or password.
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 401
}
Stack trace:
Unauthorized: Invalid username or password.
    at AuthService.login (\services\authService.ts:105:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async AuthController.login (\controllers\authController.ts:31:40)
2024-12-19T19:29:03.703Z [ERROR]: Error: Route not found
Meta: {
  "url": "/api/auth/logindfg",
  "method": "POST",
  "statusCode": 404
}
Stack trace:
Not Found: Route not found
    at notFoundMiddleware (\middlewares\notFoundMiddleware.ts:10:7)
2024-12-19T19:29:08.086Z [ERROR]: Error: Invalid username or password.
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 401
}
Stack trace:
Unauthorized: Invalid username or password.
    at AuthService.login (\services\authService.ts:105:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async AuthController.login (\controllers\authController.ts:31:40)
2024-12-19T19:29:18.891Z [ERROR]: Error: Invalid username or password.
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 401
}
Stack trace:
Unauthorized: Invalid username or password.
    at AuthService.login (\services\authService.ts:140:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async AuthController.login (\controllers\authController.ts:31:40)
2024-12-19T19:29:20.108Z [ERROR]: Error: Invalid username or password.
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 401
}
Stack trace:
Unauthorized: Invalid username or password.
    at AuthService.login (\services\authService.ts:140:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async AuthController.login (\controllers\authController.ts:31:40)
2024-12-19T19:29:20.966Z [ERROR]: Error: Invalid username or password.
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 401
}
Stack trace:
Unauthorized: Invalid username or password.
    at AuthService.login (\services\authService.ts:140:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async AuthController.login (\controllers\authController.ts:31:40)
2024-12-19T19:29:22.026Z [ERROR]: Error: Invalid username or password.
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 401
}
Stack trace:
Unauthorized: Invalid username or password.
    at AuthService.login (\services\authService.ts:140:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async AuthController.login (\controllers\authController.ts:31:40)
2024-12-19T19:29:23.234Z [ERROR]: Error: Invalid username or password.
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 401
}
Stack trace:
Unauthorized: Invalid username or password.
    at AuthService.login (\services\authService.ts:140:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async AuthController.login (\controllers\authController.ts:31:40)
2024-12-19T19:29:24.359Z [ERROR]: Error: Account is locked. Try again later.
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 429
}
Stack trace:
Too Many Requests Error: Account is locked. Try again later.
    at AuthService.login (\services\authService.ts:130:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async AuthController.login (\controllers\authController.ts:31:40)
2024-12-19T19:29:26.173Z [ERROR]: Error: Account is locked. Try again later
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 429
}
Stack trace:
Too Many Requests Error: Account is locked. Try again later
    at AuthService.login (\services\authService.ts:116:12)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async AuthController.login (\controllers\authController.ts:31:40)
2024-12-19T19:29:27.595Z [ERROR]: Error: Account is locked. Try again later
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 429
}
Stack trace:
Too Many Requests Error: Account is locked. Try again later
    at AuthService.login (\services\authService.ts:116:12)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async AuthController.login (\controllers\authController.ts:31:40)
2024-12-19T19:29:29.038Z [ERROR]: Error: Account is locked. Try again later
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 429
}
Stack trace:
Too Many Requests Error: Account is locked. Try again later
    at AuthService.login (\services\authService.ts:116:12)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async AuthController.login (\controllers\authController.ts:31:40)
2024-12-19T19:29:30.624Z [ERROR]: Error: Account is locked. Try again later
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 429
}
Stack trace:
Too Many Requests Error: Account is locked. Try again later
    at AuthService.login (\services\authService.ts:116:12)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async AuthController.login (\controllers\authController.ts:31:40)
2024-12-19T19:29:35.456Z [ERROR]: Error: Account is locked. Try again later
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 429
}
Stack trace:
Too Many Requests Error: Account is locked. Try again later
    at AuthService.login (\services\authService.ts:116:12)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async AuthController.login (\controllers\authController.ts:31:40)
2024-12-19T19:31:45.978Z [ERROR]: Error: Invalid username or password.
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 401
}
Stack trace:
Unauthorized: Invalid username or password.
    at AuthService.login (\services\authService.ts:140:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async AuthController.login (\controllers\authController.ts:31:40)
2024-12-20T22:29:32.286Z [ERROR]: Error: Invalid username or password.
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 401
}
Stack trace:
Unauthorized: Invalid username or password.
    at AuthService.login (\services\authService.ts:105:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async AuthController.login (\controllers\authController.ts:31:40)
2024-12-20T22:29:44.518Z [ERROR]: Error: Invalid username or password.
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 401
}
Stack trace:
Unauthorized: Invalid username or password.
    at AuthService.login (\services\authService.ts:105:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async AuthController.login (\controllers\authController.ts:31:40)
2024-12-20T22:30:11.458Z [ERROR]: Error: Account is locked. try again later.
Meta: {
  "url": "/api/auth/login",
  "method": "POST",
  "statusCode": 429
}
Stack trace:
Too Many Requests Error: Account is locked. try again later.
    at \utils\limiter.ts:71:12
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
2024-12-20T22:35:13.308Z [ERROR]: Error: Route not found
Meta: {
  "url": "/api/contacts/contact/6765d200e7022695766193fe",
  "method": "GET",
  "statusCode": 404
}
Stack trace:
Not Found: Route not found
    at notFoundMiddleware (\middlewares\notFoundMiddleware.ts:10:7)
2024-12-20T22:35:27.678Z [ERROR]: Error: Route not found
Meta: {
  "url": "/api/contacts/contact/6765d200e7022695766193fe",
  "method": "GET",
  "statusCode": 404
}
Stack trace:
Not Found: Route not found
    at notFoundMiddleware (\middlewares\notFoundMiddleware.ts:10:7)
2024-12-20T23:24:21.852Z [ERROR]: Error: Invalid Token: TokenExpiredError: jwt expired
Meta: {
  "url": "/api/contacts/contacts-by-location?location=maman&page=2&limit=3",
  "method": "GET",
  "statusCode": 403
}
Stack trace:
Forbidden Error: Invalid Token: TokenExpiredError: jwt expired
    at userAuth (\middlewares\userAuth.ts:44:8)
2024-12-20T23:25:00.879Z [ERROR]: Error: No contacts found for the location: undefined
Meta: {
  "url": "/api/contacts/contacts-by-location?service=agent&page=2&limit=3",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the location: undefined
    at ContactService.selectByLocation (\services\contactService.ts:101:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByLocation (\controllers\contactController.ts:72:35)
2024-12-20T23:25:29.629Z [ERROR]: Error: No contacts found for the location: undefined
Meta: {
  "url": "/api/contacts/contacts-by-location?service=agent&page=2&limit=3",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the location: undefined
    at ContactService.selectByLocation (\services\contactService.ts:101:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByLocation (\controllers\contactController.ts:72:35)
2024-12-21T00:07:58.118Z [ERROR]: Error: No contacts found for the location: location
Meta: {
  "url": "/api/contacts/contacts-by-location?location=maman",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the location: location
    at ContactService.selectByLocation (\services\contactService.ts:101:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByLocation (\controllers\contactController.ts:75:35)
2024-12-21T00:08:42.713Z [ERROR]: Error: No contacts found for the location: service
Meta: {
  "url": "/api/contacts/contacts-by-location?service=agent&page=2&limit=3",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the location: service
    at ContactService.selectByLocation (\services\contactService.ts:101:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByLocation (\controllers\contactController.ts:75:35)
2024-12-21T00:09:04.696Z [ERROR]: Error: No contacts found for the location: location
Meta: {
  "url": "/api/contacts/contacts-by-location?page=2&limit=3&location=maman",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the location: location
    at ContactService.selectByLocation (\services\contactService.ts:101:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByLocation (\controllers\contactController.ts:75:35)
2024-12-21T00:11:27.871Z [ERROR]: Error: No contacts found for the location: location
Meta: {
  "url": "/api/contacts/contacts-by-location?page=2&limit=3&location=maman",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the location: location
    at ContactService.selectByLocation (\services\contactService.ts:101:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByLocation (\controllers\contactController.ts:74:35)
2024-12-21T00:12:09.912Z [ERROR]: Error: No contacts found for the location: location
Meta: {
  "url": "/api/contacts/contacts-by-location?page=2&limit=3&location=maman",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the location: location
    at ContactService.selectByLocation (\services\contactService.ts:101:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByLocation (\controllers\contactController.ts:75:35)
2024-12-21T00:13:14.442Z [ERROR]: Error: No contacts found for the location: location
Meta: {
  "url": "/api/contacts/contacts-by-location?page=2&limit=3&location=maman",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the location: location
    at ContactService.selectByLocation (\services\contactService.ts:103:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByLocation (\controllers\contactController.ts:75:35)
2024-12-21T00:14:21.035Z [ERROR]: Error: An unknown error occurred
Meta: {
  "url": "/api/contacts/contacts-by-location?page=2&limit=3&location=maman",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
ReferenceError: location is not defined
    at ContactService.selectByLocation (\services\contactService.ts:121:64)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByLocation (\controllers\contactController.ts:75:35)
2024-12-21T00:14:26.663Z [ERROR]: Error: An unknown error occurred
Meta: {
  "url": "/api/contacts/contacts-by-location?page=2&limit=3&location=maman",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
ReferenceError: location is not defined
    at ContactService.selectByLocation (\services\contactService.ts:121:64)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByLocation (\controllers\contactController.ts:75:35)
2024-12-21T00:14:52.994Z [ERROR]: Error: No contacts found for the location: location
Meta: {
  "url": "/api/contacts/contacts-by-location?page=2&limit=3&location=maman",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the location: location
    at ContactService.selectByLocation (\services\contactService.ts:103:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByLocation (\controllers\contactController.ts:74:35)
2024-12-21T00:26:05.385Z [ERROR]: Error: An unknown error occurred
Meta: {
  "url": "/api/contacts/contacts-by-location?page=2&limit=3&location=maman",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
ReferenceError: location is not defined
    at ContactService.selectByLocation (\services\contactService.ts:103:56)
    at ContactController.selectByLocation (\controllers\contactController.ts:81:54)
    at userAuth (\middlewares\userAuth.ts:36:3)
2024-12-21T00:26:51.305Z [ERROR]: Error: An unknown error occurred
Meta: {
  "url": "/api/contacts/contacts-by-location?page=2&limit=3&location=maman",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
ReferenceError: location is not defined
    at ContactService.selectByLocation (\services\contactService.ts:103:56)
    at ContactController.selectByLocation (\controllers\contactController.ts:81:54)
    at userAuth (\middlewares\userAuth.ts:36:3)
2024-12-21T14:03:19.197Z [ERROR]: Error: No contacts found for the location: maman
Meta: {
  "url": "/api/contacts/contacts-by-location?page=2&limit=3&location=maman",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the location: maman
    at ContactService.selectByLocation (\services\contactService.ts:119:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByLocation (\controllers\contactController.ts:72:35)
2024-12-21T14:03:25.179Z [ERROR]: Error: No contacts found for the location: maman
Meta: {
  "url": "/api/contacts/contacts-by-location?page=2&limit=3&location=maman",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the location: maman
    at ContactService.selectByLocation (\services\contactService.ts:119:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByLocation (\controllers\contactController.ts:72:35)
2024-12-21T14:04:59.359Z [ERROR]: Error: No contacts found for the location: maman
Meta: {
  "url": "/api/contacts/contacts-by-location?page=2&limit=3&location=maman",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the location: maman
    at ContactService.selectByLocation (\services\contactService.ts:121:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByLocation (\controllers\contactController.ts:72:35)
2024-12-21T14:05:58.831Z [ERROR]: Error: No contacts found for the location: maman
Meta: {
  "url": "/api/contacts/contacts-by-location?page=2&limit=3&location=maman",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the location: maman
    at ContactService.selectByLocation (\services\contactService.ts:121:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByLocation (\controllers\contactController.ts:73:35)
2024-12-21T14:07:00.749Z [ERROR]: Error: No contacts found for the location: maman
Meta: {
  "url": "/api/contacts/contacts-by-location?page=2&limit=3&location=maman",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the location: maman
    at ContactService.selectByLocation (\services\contactService.ts:121:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByLocation (\controllers\contactController.ts:73:35)
2024-12-21T14:07:12.181Z [ERROR]: Error: No contacts found for the location: maman
Meta: {
  "url": "/api/contacts/contacts-by-location?page=2&limit=3&location=maman",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the location: maman
    at ContactService.selectByLocation (\services\contactService.ts:121:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByLocation (\controllers\contactController.ts:73:35)
2024-12-21T14:15:48.309Z [ERROR]: Error: No contacts found for the location: test record 10
Meta: {
  "url": "/api/contacts/contacts-by-company?limit=3&page=1&company=test%20record%2010",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the location: test record 10
    at ContactService.selectByLocation (\services\contactService.ts:103:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByCompany (\controllers\contactController.ts:101:34)
2024-12-21T14:16:35.343Z [ERROR]: Error: No contacts found for the location: "test record 10"
Meta: {
  "url": "/api/contacts/contacts-by-company?limit=3&page=1&company=%22test%20record%2010%22",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the location: "test record 10"
    at ContactService.selectByLocation (\services\contactService.ts:103:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByCompany (\controllers\contactController.ts:101:34)
2024-12-21T14:17:57.840Z [ERROR]: Error: No contacts found for the location: test%record%10
Meta: {
  "url": "/api/contacts/contacts-by-company?page=1&limit=3&company=test%record%10",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the location: test%record%10
    at ContactService.selectByLocation (\services\contactService.ts:103:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByCompany (\controllers\contactController.ts:101:34)
2024-12-21T14:18:46.012Z [ERROR]: Error: No contacts found for the location: test record 10
Meta: {
  "url": "/api/contacts/contacts-by-company?page=1&limit=3&company=test%20record%2010",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the location: test record 10
    at ContactService.selectByLocation (\services\contactService.ts:103:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByCompany (\controllers\contactController.ts:101:34)
2024-12-21T14:19:31.502Z [ERROR]: Error: No contacts found for the location: test record 10
Meta: {
  "url": "/api/contacts/contacts-by-company?page=1&limit=3&company=test%20record%2010",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the location: test record 10
    at ContactService.selectByLocation (\services\contactService.ts:103:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByCompany (\controllers\contactController.ts:101:34)
2024-12-21T14:19:55.940Z [ERROR]: Error: No contacts found for the location: test record 10
Meta: {
  "url": "/api/contacts/contacts-by-company?page=1&limit=3&company=test%20record%2010",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the location: test record 10
    at ContactService.selectByLocation (\services\contactService.ts:103:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByCompany (\controllers\contactController.ts:101:34)
2024-12-21T14:19:57.287Z [ERROR]: Error: No contacts found for the location: test record 10
Meta: {
  "url": "/api/contacts/contacts-by-company?page=1&limit=3&company=test%20record%2010",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the location: test record 10
    at ContactService.selectByLocation (\services\contactService.ts:103:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByCompany (\controllers\contactController.ts:101:34)
2024-12-21T14:21:36.421Z [ERROR]: Error: Invalid Token: TokenExpiredError: jwt expired
Meta: {
  "url": "/api/contacts/contacts-by-location?limit=3&location=maman&page=1",
  "method": "GET",
  "statusCode": 403
}
Stack trace:
Forbidden Error: Invalid Token: TokenExpiredError: jwt expired
    at userAuth (\middlewares\userAuth.ts:44:8)
2024-12-21T15:10:24.365Z [ERROR]: Error: Route not found
Meta: {
  "url": "/api/contacts/find-contact-by-email?email=person5email@mail.com",
  "method": "GET",
  "statusCode": 404
}
Stack trace:
Not Found: Route not found
    at notFoundMiddleware (\middlewares\notFoundMiddleware.ts:10:7)
2024-12-21T15:11:15.504Z [ERROR]: Error: Contact not found.
Meta: {
  "url": "/api/contacts/find-contact-by-email?email=person5email@mail.com",
  "method": "GET",
  "statusCode": 409
}
Stack trace:
Conflict Error: Contact not found.
    at ContactService.getContactByEmail (\services\contactService.ts:149:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.getContactByEmail (\controllers\contactController.ts:127:20)
2024-12-21T15:11:47.290Z [ERROR]: Error: Contact not found.
Meta: {
  "url": "/api/contacts/find-contact-by-email?email=person5email@mail.com",
  "method": "GET",
  "statusCode": 409
}
Stack trace:
Conflict Error: Contact not found.
    at ContactService.getContactByEmail (\services\contactService.ts:149:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.getContactByEmail (\controllers\contactController.ts:127:20)
2024-12-21T15:18:18.549Z [ERROR]: Error: Route not found
Meta: {
  "url": "/api/contacts/contacts-by-service?limit=3&page=1&service=airline",
  "method": "GET",
  "statusCode": 404
}
Stack trace:
Not Found: Route not found
    at notFoundMiddleware (\middlewares\notFoundMiddleware.ts:10:7)
2024-12-21T15:18:27.307Z [ERROR]: Error: Route not found
Meta: {
  "url": "/api/contacts/search-by-service?limit=3&page=1&service=airline",
  "method": "GET",
  "statusCode": 404
}
Stack trace:
Not Found: Route not found
    at notFoundMiddleware (\middlewares\notFoundMiddleware.ts:10:7)
2024-12-21T15:18:49.824Z [ERROR]: Error: Route not found
Meta: {
  "url": "/api/contacts/search-by-service?limit=3&page=1&service=airline",
  "method": "GET",
  "statusCode": 404
}
Stack trace:
Not Found: Route not found
    at notFoundMiddleware (\middlewares\notFoundMiddleware.ts:10:7)
2024-12-21T15:19:09.852Z [ERROR]: Error: Invalid Token: TokenExpiredError: jwt expired
Meta: {
  "url": "/api/contacts/search-contacts-by-service?limit=3&page=1&service=airline",
  "method": "GET",
  "statusCode": 403
}
Stack trace:
Forbidden Error: Invalid Token: TokenExpiredError: jwt expired
    at userAuth (\middlewares\userAuth.ts:44:8)
2024-12-21T15:20:40.136Z [ERROR]: Error: No contacts found for the company: Agent
Meta: {
  "url": "/api/contacts/search-contacts-by-service?limit=3&page=1&service=Agent",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the company: Agent
    at ContactService.selectByService (\services\contactService.ts:154:10)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByService (\controllers\contactController.ts:105:34)
2024-12-21T15:20:45.101Z [ERROR]: Error: No contacts found for the company: gent
Meta: {
  "url": "/api/contacts/search-contacts-by-service?limit=3&page=1&service=gent",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the company: gent
    at ContactService.selectByService (\services\contactService.ts:154:10)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByService (\controllers\contactController.ts:105:34)
2024-12-21T15:21:31.755Z [ERROR]: Error: No contacts found for the service: gent
Meta: {
  "url": "/api/contacts/search-contacts-by-service?limit=3&page=1&service=gent",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the service: gent
    at ContactService.selectByService (\services\contactService.ts:142:10)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByService (\controllers\contactController.ts:105:34)
2024-12-21T15:21:36.996Z [ERROR]: Error: No contacts found for the service: agent
Meta: {
  "url": "/api/contacts/search-contacts-by-service?limit=3&page=1&service=agent",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the service: agent
    at ContactService.selectByService (\services\contactService.ts:142:10)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByService (\controllers\contactController.ts:105:34)
2024-12-21T15:21:50.023Z [ERROR]: Error: No contacts found for the service: airline
Meta: {
  "url": "/api/contacts/search-contacts-by-service?limit=3&page=1&service=airline",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the service: airline
    at ContactService.selectByService (\services\contactService.ts:142:10)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByService (\controllers\contactController.ts:105:34)
2024-12-21T15:21:55.962Z [ERROR]: Error: No contacts found for the service: airline
Meta: {
  "url": "/api/contacts/search-contacts-by-service?limit=3&page=1&service=airline",
  "method": "GET",
  "statusCode": 500
}
Stack trace:
Internal Error: No contacts found for the service: airline
    at ContactService.selectByService (\services\contactService.ts:142:10)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ContactController.selectByService (\controllers\contactController.ts:105:34)
